lib/livebook/hubs/team.ex

```diff
@@ -147,9 +147,6 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Team do
       team.offline ->
         "You are running an offline Workspace for deployment. You cannot modify its settings."
```

Comment:
Should we use the opportunity and also move offline mode to the banner? Keeping this one only about connection status?